custom/icds_reports/templates/icds_reports/icds_app/pdf/issnip_monthly_register.html

```diff
@@ -105,8 +105,8 @@ <h2>3. Total Population as per Anganwadi Survey Register</h2>
                         <table>
                             <thead>
                                 <tr>
-                                    <th>Infants (0- 6 Months)</th>
-                                    <th>Children(6 -36 Months)</th>
+                                    <th>Infants (0-6 Months)</th>
+                                    <th>Children(6-36 Months)</th>
```

Comment:
Should this be formatted like `Children (6-36 Month)` (space between "Children" and "(") ?